x-pack/plugins/security_solution/public/common/mock/global_state.ts

```diff
@@ -164,6 +169,36 @@ export const mockGlobalState: State = {
       },
     },
   },
+  ueba: {
+    page: {
+      queries: {
+        [uebaModel.UebaTableType.riskScore]: {
+          activePage: 0,
+          limit: 10,
+          sort: { field: RiskScoreFields.riskScore, direction: Direction.desc },
+        },
+      },
+    },
+    details: {
+      queries: {
+        [uebaModel.UebaTableType.hostRules]: {
+          activePage: 0,
+          limit: 10,
+          sort: { field: HostRulesFields.riskScore, direction: Direction.desc },
+        },
+        [uebaModel.UebaTableType.hostTactics]: {
+          activePage: 0,
+          limit: 10,
+          sort: { field: HostTacticsFields.riskScore, direction: Direction.desc },
+        },
+        [uebaModel.UebaTableType.userRules]: {
```

Comment:
this is where the bug is.. more like an unfinished feature. but there are multiple user rule tables per host, so this needs to be an array of `userRules` tables or all the tables will share pagination and sorting